test/python/utils.py

```diff
@@ -255,3 +283,106 @@ def run_command_and_get_output(cmd, cwd=None, env=None):
     print(stdout)
 
     return process.returncode, stdout
+
+
+def find_image(docker_client, project_name):
+    for image in docker_client.images.list():
+        for t in image.tags:
+            if t.startswith(project_name):
+                return t
+
+
+def wait_for_container_start(container, timeout=10):
```

Comment:
Consider "HEALTHCHECK" section usage then you should wait for "healthy" status

Reply:
I didn't get it, can you explain?

Reply:
https://docs.docker.com/engine/reference/builder/#healthcheck

Reply:
This function waits while Tarantool instance in container starts (it waits `entering the event loop` in logs).

Reply:
I understand. But I think it's quite similar to situation then container goes to "healthy" state

Reply:
So, you suggest adding HEALTHCHECK section to `cartridge pack docker` result container?

I agree, that it's a good idea, but I sure that is should be implemented in a separated PR.